.github/workflows/build-rule-book.yaml

```diff
@@ -27,6 +31,16 @@ jobs:
       - name: Set up repository
         uses: actions/checkout@v4
 
+      - name: Generate translations for ${{ matrix.language }}
```

Comment:
ah nice, didn't think matrix could be included in the name

Reply:
Magic!